packages/app-extensions/src/formField/defaultMapping.js

```diff
@@ -17,25 +17,30 @@ export default {
   'email': editableValueFactory('email'),
   'html': editableValueFactory('text'),
   'image': editableValueFactory('document'),
+  'identifier': editableValueFactory('string'),
+  'latitude': editableValueFactory('number'),
   'login': editableValueFactory('string'),
-  'long': editableValueFactory('number'),
+  'long': editableValueFactory('string'),
+  'longitude': editableValueFactory('decimal'),
   'moneyamount': editableValueFactory('moneyamount'),
   'multi-remote-field': editableValueFactory('multi-remote'),
   'multi-select-box': editableValueFactory('multi-select'),
   'named-upload': editableValueFactory('document'),
-  'integer': editableValueFactory('number'),
-  'percent': editableValueFactory('number'),
+  'integer': editableValueFactory('integer'),
+  'percent': editableValueFactory('decimal'),
   'phone': editableValueFactory('phone'),
   'range': {
-    'number': editableValueFactory('number'),
-    'counter': editableValueFactory('number'),
+    'number': editableValueFactory('decimal'),
+    'counter': editableValueFactory('decimal'),
     'date': editableValueFactory('date-range'),
     'birthdate': editableValueFactory('date-range')
   },
   'single-remote-field': editableValueFactory('remote'),
   'search-filter': editableValueFactory('search-filter'),
+  'sorting': editableValueFactory('number'),
```

Comment:
i'm a little confused. why do we have 3 types (decimal, number and integer)? Shouldn't two be sufficient ? Integer for whole numbers and decimal for those with a floating point?